portfolio/src/main/webapp/index.html

```diff
@@ -155,7 +158,7 @@ <h3>Comments</h3>
       <div id="comment-section" class="mt-4 mb-5">
         <div class="card p-4 mb-4">
           <h4 class="text-center">Post your thoughts!</h4>
-          <form id="comment-form" action="/comments" method="POST">
+          <form id="comment-form">
```

Comment:
Just a thought: do we have to remove `action` and `method`? I know there are some people who browse without JS, maybe this would still work for them if we didn't remove it?

Reply:
Good point. I'll look into this.

Reply:
Just tested this with a JS blocker, and posting still works when you keep the action and method! Now that I think about it, it makes sense: the event.preventDefault() is not called because there is no JS, so the default runs, which is the traditional form submission. Nice.